### PR TITLE
Local Nav find overview page for subfolder pages

### LINKED ISSUE #419: LOCAL NAV: Update logic to determine if page has an overview page. Don't traverse past this folder level.

### PR DESCRIPTION
**Pull request by jsavella**
Draw local navigation from overview folder if page selected is subfolder page

Fix #419

Test URLs:
- Original: https://www.esri.com/en-us/capabilities/real-time/features/storage
- Before: https://main--esri-eds--esri.aem.live/en-us/capabilities/real-time/features/storage
- After: https://navSubHomePg--esri-eds--esri.aem.live/en-us/capabilities/real-time/features/storage

subfolderpage

### CODE REVIEW

File: eds/blocks/local-navigation/local-navigation.css [hunk 1]
```diff
@@ -153,6 +159,15 @@
...
 }
 
+.local-navigation calcite-button {
+  display: flex;
+  inline-size: 100%;
+  padding-inline: 30%;
+  align-items: center;
+  background-color: #202020;
+  padding-block: var(--space-4);
+}
```
Comment (twhite313) [inline eds/blocks/local-navigation/local-navigation.css:169]: pretty sure we don't need any of this. When I comment out all this excpet padding it works fine.

File: eds/blocks/local-navigation/local-navigation.css [hunk 2]
```diff
@@ -50,15 +50,20 @@
   align-items: center;
   text-decoration: none;
   background-color: #202020;
-  padding-inline: var(--space-8);
+  padding-inline: var(--space-6);
 }
 
-.local-navigation ul li a[aria-current='true']:hover, .local-navigation ul li.page-title a:hover {
+.local-navigation ul li a[aria-current='true']:hover, .local-navigation ul li a:hover {
```
Comment (twhite313) [inline eds/blocks/local-navigation/local-navigation.css:56]: Shorted up with `:is()` since everything is the same, just adding two selectors (true and a)

**Review by twhite313 — COMMENTED** [review]
Couple of comments. 

Build errors. Be sure to run `npm run lint` before putting in PR.

File: eds/blocks/local-navigation/local-navigation.css (hunk 1 above)
Comment (jsavella) [reply eds/blocks/local-navigation/local-navigation.css:169]: Was able to delete lines 164 and 166... otherwise mobile would break.

**Review by twhite313 — APPROVED** [review]

**Pull request merged**